content/docs/02-guides/06-natural-language-postgres.mdx

```diff
@@ -0,0 +1,683 @@
+---
+title: Natural Language Postgres
+description: Learn how to build a Next.js app that uses AI to interact with a PostgreSQL database using natural language.
```

Comment:
"Learn how to build a Next.js app that lets you talk to a PostgreSQL database in natural language" - potential to make it more direct (also below)